File: CS/WindowsFormsApplication31/CustomToggleSwitchViewInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.Utils.Drawing;
using DevExpress.XtraEditors.Repository;
using DevExpress.XtraEditors.ViewInfo;

namespace WindowsFormsApplication31 {
    public class CustomToggleSwitchViewInfo : ToggleSwitchViewInfo {
        public CustomToggleSwitchViewInfo(RepositoryItem item)
            : base(item) {
        }
        protected override BaseCheckObjectPainter CreateCheckPainter() {
            return new CustomToggleObjectPainter(this.LookAndFeel);
        }
    }
    
}

File: VB/WindowsFormsApplication31/CustomToggleSwitchViewInfo.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.Utils.Drawing
Imports DevExpress.XtraEditors.Repository
Imports DevExpress.XtraEditors.ViewInfo

Namespace WindowsFormsApplication31
    Public Class CustomToggleSwitchViewInfo
        Inherits ToggleSwitchViewInfo

        Public Sub New(ByVal item As RepositoryItem)
            MyBase.New(item)
        End Sub
        Protected Overrides Function CreateCheckPainter() As BaseCheckObjectPainter
            Return New CustomToggleObjectPainter(Me.LookAndFeel)
        End Function
    End Class

End Namespace
